apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -243,7 +243,10 @@ export default function sectionProgress(state=initialState, action) {
           scriptId = validScripts[0].id;
           break;
         default:
-         scriptId = ACCELERATED_SCRIPT_ID;
+          // Use the default script (currently Express Course) so we don't have
+          // an empty dropdown.
+          scriptId = defaultScript.id;
+          validScripts.push(defaultScript);
```

Comment:
this is starting to get a bit confusing, because now the part of the function that determines the `scriptId` is also modifying `validScripts`. As a first step, I would recommend moving the logic for `if (validScripts.length === 0) {validScripts.push(defaultScript)}` to line 226 and replacing the `default` and last `case` statement here with just
```js
default:
  scriptId = validScripts[0].id
```
and also adding a comment explaining that one section computes validScripts and the other computes scriptId.

Ideally you would extract 1 or 2 self-documenting functions like this:
```js
if (action.type === SET_VALID_SCRIPTS) {
  const validScripts = computeValidScripts(...)
  const scriptId = getDefaultScriptId(...)
}
```
but given the time constraints it seems ok not to do that now.